### PR TITLE
[12.0][MIG] procurement_auto_create_group

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests ok

**Comment by AaronHForgeFlow** [comment]
Please check travis @mreficent

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

File: procurement_auto_create_group/README.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#. Go to *Inventory / Configuration / Warehouse Management / Routes*
+   and select the route you want to change. Select the pull rule you wish
+   to change and Select 'Propagation of Procurement Group': 'Propagage'.
+   The checkbox 'Auto-create Procurement Group' will then appear and you can
+   set it if you want to procurement group to be automatically created.
+
+Usage
+=====
+
+#. Create a new procurement and make sure that it determines a pull rule
```
Comment (rousseldenis) [inline procurement_auto_create_group/README.rst]: You need to change this

File: procurement_auto_create_group/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Procurement Auto Create Group",
+    "version": "12.0.1.0.0",
+    "development_status": "Production/Stable",
+    "license": "AGPL-3",
+    "summary": "Allows to configure the system to propose automatically new "
+               "procurement groups in procurement orders.",
```
Comment (rousseldenis) [inline procurement_auto_create_group/__manifest__.py]: You need to change this as procurements does not exist anymore

File: procurement_auto_create_group/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "author": "Eficent,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
```
Comment (rousseldenis) [inline procurement_auto_create_group/__manifest__.py]: The good category is 'Warehouse'

File: procurement_auto_create_group/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,9 @@
+This module allows the system to propose automatically new procurement groups
+when procuring.
+
+This capability is important when you want to make sure that all the stock
+moves resulting from this procurement will never be mixed with moves from
```
Comment (rousseldenis) [inline procurement_auto_create_group/readme/DESCRIPTION.rst]: You need to change this (and under too)

File: procurement_auto_create_group/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#. Create a new procurement and make sure that it determines a pull rule
```
Comment (rousseldenis) [inline procurement_auto_create_group/readme/USAGE.rst]: You need to change this

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. LGTM. You need to adapt all texts to remove procurement references.

File: procurement_auto_create_group/README.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [reply procurement_auto_create_group/README.rst]: Suggestion?

File: procurement_auto_create_group/__manifest__.py (hunk 2 above)
Comment (MiquelRForgeFlow) [reply procurement_auto_create_group/__manifest__.py]: Suggestion?

File: procurement_auto_create_group/readme/DESCRIPTION.rst (hunk 4 above)
Comment (MiquelRForgeFlow) [reply procurement_auto_create_group/readme/DESCRIPTION.rst]: Suggestion?

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis hope you like the new descriptions :)

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot, everything ok :+1:

**Comment by LoisRForgeFlow** [comment]
@rousseldenis good to go?

**Comment by rousseldenis** [comment]
@lreficent What I wanted to say is that 'procurement' does not exists anymore and so, 'running procurements' has no sense (from a functionnal point of view)...

**Comment by LoisRForgeFlow** [comment]
@rousseldenis I think it does, you still "procure" material, which is quite similar. Also technically it uses the method run, so is not that bad to say run the procurement (aka run "the need for materials").

Yet it could likely be stated more clearly, so... which is your proposal?

**Comment by rousseldenis** [comment]
> I think it does, you still "procure" material, which is quite similar.

Ok, that's good for me. I just didn't want user expect to find 'procurements'... Let's merge

**Review by rousseldenis — APPROVED** [review]

**Pull request closed without merging**